src/wp-includes/option.php

```diff
@@ -2037,6 +2037,25 @@ function get_network_option( $network_id, $option, $default_value = false ) {
 	 */
 	$pre = apply_filters( "pre_site_option_{$option}", false, $option, $network_id, $default_value );
 
+	/**
+	 * Filters the value of an existing network options before it is retrieved.
+	 *
+	 * Returning a truthy value from the filter will effectively short-circuit retrieval
+	 * and return the passed value instead.
```

Comment:
```suggestion
	 * Returning a value other than false from the filter will short-circuit retrieval
	 * and return that value instead.
```

It's a strict check on `false`, so some none truthy values (such as an empty array) would still short-circuit retrieval.  I think we should use the same langauge as the filter above.

Reply:
Noting that the `pre_option` filter also needs to get fixed up for this as well since it too uses `truthy`.